physics/archimedes_principle.py

```diff
@@ -10,8 +10,8 @@
 """
 
 
-# Acceleration Constant on Earth (unit m/s^2)
-g = 9.80665
+# Acceleration Constant on Earth (unit m/s^2) imported from scipy
+from scipy.constants import g
```

Comment:
Please revert this change.  Let's not create a dependency on all of scipy just to set a constant.